### PR TITLE
Added app.run and secret key

### PR DESCRIPTION
**Pull request by euab**
The app.run will run the app. I'm not sure whether or not you need it for what you are doing but it is pretty standard in Flask to use it so I'm assuming it was left out.

The SECRET_KEY means that the flask app can do more because of the encryption that comes with it.

### CODE REVIEW

**Comment by alvierahman90** [comment]
have you tested it?

**Comment by euab** [comment]
About to

**Comment by alvierahman90** [comment]
cool, I'll check it tomorrow

**Comment by euab** [comment]
You'll have to try the config file has been ignored and I don't wanna set up Gunicorn

File: uploadr.py [hunk 1]
```diff
@@ -44,3 +46,6 @@ def css(filename=None):
 @app.route('/fonts/<filename>')
 def fonts(filename=None):
     return flask.send_from_directory('fonts', filename)
+
+app.debug = True
```
Comment (alvierahman90) [inline uploadr.py]: how dare you assume my program may ever have a bug?!?! /s

**Pull request merged**